client/container_wait_test.go

```diff
@@ -117,6 +118,39 @@ func TestContainerWaitProxyInterruptLong(t *testing.T) {
 	}
 }
 
+type cancelReader struct{}
+
+func (cancelReader) Read(p []byte) (n int, err error) {
+	return 0, context.Canceled
+}
+
+func TestContainerWaitContextCancelled(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	client := &Client{
+		version: "1.30",
+		client: newMockClient(func(req *http.Request) (*http.Response, error) {
+			return &http.Response{
+				StatusCode: http.StatusOK,
+				Body:       io.NopCloser(TestClient_ContainerWait_cancelReader{}),
```

Comment:
Looks like you missed this one when renaming;

```
client/container_wait_test.go:136:30: undefined: TestClient_ContainerWait_cancelReader
```

Reply:
🤦‍♂️ Sorry... Fixed. Happens when you think "oh let's do that quickly and forget about that IDE"

Reply:
Happens to me all the time! 😂 no harm done, that's where review and CI is for!